rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+    @person = person
+    person.rentals << self
+  end
+
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ OPTIONAL] For better code organization, it will be nice to move these variables to the top of the class.